contracts/token/ERC777/ERC777.sol

```diff
@@ -352,6 +352,7 @@ contract ERC777 is Context, IERC777, IERC20 {
     )
         internal
     {
```

Comment:
The description for this function can be improved.

Reply:
I added a note in https://github.com/OpenZeppelin/openzeppelin-contracts/issues/1928